Tools/CISupport/ews-build/master.cfg

```diff
@@ -59,7 +59,7 @@ if is_test_mode_enabled:
     c['buildbotURL'] = 'http://localhost:8010/'
     c['db_url'] = 'sqlite:///state.sqlite?serialize_access=1'
 else:
-    c['buildbotURL'] = 'https://ews-build.webkit{}.org/'.format(custom_suffix)
+    c['buildbotURL'] = utils.load_password('MASTERCFG_BUILDBOT_URL', default='https://ews-build.webkit{}.org/'.format(custom_suffix))
```

Comment:
Let's use f-string while we are at it. In general we want to convert all the .format to f-string.

Nit: `MASTERCFG_BUILDBOT_URL` can simply be `BUILDBOT_URL`